akka-http-core/src/main/mima-filters/10.1.9.backwards.excludes/pr-2708-auto-generated.excludes

```diff
@@ -0,0 +1,21 @@
+# Autogenerated MiMa filters, please pull branch and check validity, and add comments why they are needed etc.
+# Don't merge like this.
+
+# Incompatibilities against 10.1.6, 10.1.7, 10.1.8, 10.1.9
```

Comment:
I really like this. I wonder if it wouldn't make more sense to have it in the validation report rather than appended to the PR, though.